proposals/dao/vip_11.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import config from '../../scripts/config';
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+const { STARTING_ORACLE_PRICE, ORACLE_PERIOD_START_TIME, MONTHLY_CHANGE_RATE_BASIS_POINTS } = config;
+
+/*
+
+Volt Protocol Improvement Proposal #11
+
+Description: Patch in a new system oracle and repoint all PSM's to it
+
+Steps:
+  1 - Deploy Volt System Oracle
+  2 - Point Oracle Pass Through to New Oracle
+
+*/
+
+const vipNumber = '11';
+
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const VoltSystemOracleFactory = await ethers.getContractFactory('VoltSystemOracle');
+
+  const voltSystemOracle = await VoltSystemOracleFactory.deploy(
+    MONTHLY_CHANGE_RATE_BASIS_POINTS,
+    ORACLE_PERIOD_START_TIME,
+    STARTING_ORACLE_PRICE
+  );
+  await voltSystemOracle.deployed();
+
+  console.log(`Volt System Oracle ${voltSystemOracle.address}`);
+
+  console.log(`Deployed volt system oracle VIP-${vipNumber}`);
+  return {
+    voltSystemOracle
+  };
+};
+
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {};
+
+// Tears down any changes made in setup() that need to be
+// cleaned up before doing any validation checks.
+const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {};
+
+// Run any validations required on the vip using mocha or console logging
+// IE check balances, check state of contracts, etc.
+const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  const { oraclePassThrough, voltSystemOracle, feiPriceBoundPSM, usdcPriceBoundPSM, timelockController } = contracts;
```

Comment:
i think we can remove the contracts we aren't using here `feiPriceBoundPSM`, `usdcPriceboundPSM`

Reply:
done